Find a functionally equivalent code:android.widget.SuggestionsAdapter.getDrawable:COMMENT
Method Modifier: private     
Comment:/**
 * Gets a drawable by URI, without using the cache.
 *
 * @return A drawable, or {@code null} if the drawable could not be loaded.
 */

@@ -5,11 +5,11 @@
             // Load drawables through Resources, to get the source density information
             OpenResourceIdResult r = mProviderContext.getContentResolver().getResourceId(uri);
             try {
-                return r.r.getDrawable(r.id);
+                return r.r.getDrawable(r.id, mContext.getTheme());
             } catch (Resources.NotFoundException ex) {
                 throw new FileNotFoundException("Resource does not exist: " + uri);
             }

